install/kubernetes/cilium/charts/operator/values.yaml

```diff
@@ -1,4 +1,11 @@
-image: operator
+# Include the cilium-operator Deployment
+enabled: true
+
+image:
+  repository: cilium-operator
```

Comment:
`s/cilium-operator/operator`

Reply:
According to @errordeveloper this is supposed to change to the cilium-operator-generic container, so I am receiving conflicting information.

Reply:
Right now it's `operator` / `operator-generic`:

https://hub.docker.com/repository/docker/cilium/operator
https://hub.docker.com/repository/docker/cilium/operator-generic